### PR TITLE
improved model with batchnormalization layers

### PR DESCRIPTION
**Pull request by brianmcera**
This is just a first pull request with the initial A2C model and tuning!

### CODE REVIEW

File: src/A2C_agent.py~ [hunk 1]
```diff
@@ -12,11 +12,11 @@ import A2C_model
```
Comment (christopher-w-murphy) [inline src/A2C_agent.py~]: You can add *.py~ to your .gitignore so that these backup files aren't pushed

File: src/A2C_model.py [hunk 2]
```diff
@@ -13,25 +13,30 @@ def call(self, logits, **kwargs):
 class Model(Model):
     def __init__(self, ac_dim):
         super(Model, self).__init__()
-        self.conv1 = Conv2D(16, 3, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv2 = Conv2D(16, 3, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv3 = Conv2D(32, 3, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv1 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv2 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
```
Comment (christopher-w-murphy) [inline src/A2C_model.py]: Consider passing these numbers as arguments to your class. It will reduce/eliminate the need to make changes like this later. But don't overthink it. If you can move faster the way you're currently doing it, then keep doing it and we'll worry about this later

File: src/A2C_model.py [hunk 3]
```diff
@@ -13,25 +13,30 @@ def call(self, logits, **kwargs):
...
         self.d1 = Dense(256, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.d2 = Dense(50, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.d3 = Dense(50, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.d2 = Dense(100, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.d3 = Dense(100, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
         self.d4 = Dense(256, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.d5 = Dense(50, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.d6 = Dense(50, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.d5 = Dense(100, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.d6 = Dense(100, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
```
Comment (christopher-w-murphy) [inline src/A2C_model.py]: Same as above, consider passing as argument

File: src/A2C_model.py~ [hunk 4]
```diff
@@ -13,25 +13,30 @@ class ProbabilityDistribution(Model):
 class Model(Model):
     def __init__(self, ac_dim):
         super(Model, self).__init__()
-        self.conv1 = Conv2D(16, 3, activation='sigmoid', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv2 = Conv2D(16, 3, activation='sigmoid', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv3 = Conv2D(32, 3, activation='sigmoid', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv1 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv2 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
```
Comment (christopher-w-murphy) [inline src/A2C_model.py~]: Maybe even pass the activation as an argument (maybe have a dictionary of kwargs if there are a lot of arguments)

File: src/A2C_model.py~ [hunk 5]
```diff
@@ -40,22 +45,27 @@ class Model(Model):
...
         flattened = self.flatten(i)
+
         # actor dense layers
         x = self.d1(flattened)
-        x = self.batchnormalization2(x)
         x = self.d2(x)
-        #x = self.dropout(x)
+        x = self.batchnormalization4(x)
         x = self.d3(x)
+        #x = self.dropout2(x)
```
Comment (christopher-w-murphy) [inline src/A2C_model.py~]: It might be okay for now as you're still working through it, but in the final version of your code don't include commented out lines like this

**Review by christopher-w-murphy — APPROVED** [review]
